### DISCUSSION TITLE
2.12.0 Removed CPU Boost Mode settings?

### DISCUSSION THREAD

**Discussion by LuckyV1** [BartoszCichecki/LenovoLegionToolkit#674] [Q&A] [answered]
Hello, I would like to thank the creators for this great tool. The program works very well on my 16ACHg6. I use it a lot.
I am just wondering why is the CPU Boost Mode setting removed? I find it very useful.

**Comment by BartoszCichecki** [comment, marked as answer]
It is a set-and-forget setting that is accessible from Windows UI, so there was no point in maintaining the code in LLT. There are also other great tools that give access to other settings of Windows power plans like [Power Settings Explorer](https://forums.guru3d.com/threads/windows-power-plan-settings-explorer-utility.416058), where you can modify all setting of a power plan instead of just this single one.
